### PR TITLE
Fix daily graph to collect extra 6 days of data for moving ave

### PR DESCRIPTION
**Pull request by LPeterson18**
Fix api to actually collect extra 6 days of data for seven day moving average graph.

### CODE REVIEW

File: src/indexApi.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ async function main(): Promise<void> {
     }
     const { start, end, appId, pluginIds } = analyticsQuery
     const timePeriod = analyticsQuery.timePeriod.toLowerCase()
-    const queryOffset = timePeriod === 'day' ? SIX_DAYS_IN_MINUTES : 0
+    const queryOffset = timePeriod.includes('day') ? SIX_DAYS_IN_MINUTES : 0
```
Comment (algtan) [inline src/indexApi.ts:101]: Constant should probably be renamed to `SIX_DAYS_IN_SECONDS`. The calculation on line 11 is 6 days x 24 hours/day x 60 minutes / hour x 60 seconds / minutes = 518400 seconds.

**Review by algtan — CHANGES_REQUESTED** [review]
One minor suggestion: rename the constant `SIX_DAYS_IN_MINUTES` to `SIX_DAYS_IN_SECONDS`.

File: src/indexApi.ts (hunk 1 above)
Comment (LPeterson18) [reply src/indexApi.ts]: Fixed

**Comment by LPeterson18** [comment]
Changed constant name

**Review by algtan — COMMENTED** [review]
Looks good, I will reassign to @thehobbit85  for review.

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**